assistedInstallerService.py

```diff
@@ -115,13 +125,13 @@ def _customized_pod_persistent(self) -> Dict[str, str]:
             logger.error(f"Failed to load yaml: {self.podFile}")
             sys.exit(-1)
 
-        saas_version = "v2.18.4"
+        saas_version = "v2.29.0"
```

Comment:
can this string be removed? Aren't we setting this with "SAAS_VERSION" now?

Reply:
Ah, right, it should be.  Good catch!

Reply:
Done.